addons/crm/crm_lead.py

```diff
@@ -59,6 +59,11 @@ class crm_lead(format_address, osv.osv):
     _inherit = ['mail.thread', 'ir.needaction_mixin', 'utm.mixin']
     _mail_mass_mailing = _('Leads / Opportunities')
 
+    def _get_default_probability(self, cr, uid, context=None):
+        """ Gives default probability """
+        stage_id = self._get_default_stage_id(cr, uid, context=None)
+        return self.pool['crm.stage'].browse(cr, uid, stage_id, context=context).probability
```

Comment:
maybe `return self.pool['crm.stage'].browse(cr, uid, stage_id, context=context).probability if stage_id else 10` to be very safe, but probably not really necessary